tests/test-datajson.bats

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env batsPI
+
+load globals
+load test_helper
+
+@test "datajson_harvest extension is loaded" {
+  test_extension_loaded datajson_harvest
+}
+
+@test "data.json harvest source is created and harvested successfully" {
```

Comment:
This seems like a pretty big test, are there any parts that can be split out into reusable functions for future tests?  Or is it all pretty specific to testing this feature?

Reply:
It is long test because the final UI test is depending the datasets harvested from previous actions. I agree the harvesting part can be separated into another function, but it is used only once. If other tests also use harvested datasets, then the DRY principle will kick in.